manifests/params.pp

```diff
@@ -7,6 +7,36 @@
   $epel_includepkgs = absent
   $epel_exclude = absent
 
+  # ELREPO Settings.
+  # ELREPO has four repos, "main", "testing", "kernel" and "extras".
+  $elrepo_main_url = "http://elrepo.org/linux/elrepo/el${::operatingsystemmajrelease}/${::architecture}"
```

Comment:
I don't like setting both 'baseurl' and 'mirrorlist' in yum repo configs. I think a better approach here would be to set all the ```_url``` variables to ```absent```, and only leave the mirrorlist setting for each repo.